.github/workflows/release-automation.yml

```diff
@@ -152,6 +152,11 @@ jobs:
           gpg_private_key: ${{ secrets.GPG_KEY_CONTENTS }}
           passphrase: ${{ secrets.GPG_KEY_PASSPHRASE }}
 
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
```

Comment:
Change this to `uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2`

And then it'll be good to go

Reply:
Done.